apisix/discovery/dns/init.lua

```diff
@@ -64,14 +64,14 @@ end
 function _M.init_worker()
     local local_conf = config_local.local_conf()
     local servers = local_conf.discovery.dns.servers
-
+    local resolvconf = local_conf.discovery.dns.resolvconf
     local default_order = {"last", "SRV", "A", "AAAA", "CNAME"}
     local order = core.table.try_read_attr(local_conf, "discovery", "dns", "order")
     order = order or default_order
 
     local opts = {
         hosts = {},
-        resolvConf = {},
+        resolvConf = resolvconf,
```

Comment:
Use `resolv_conf` to replace all variables

Reply:
okay except for the `resolveConf` inside opts because that field is in the lua-resty-dns.

Reply:
done